README.md

```diff
@@ -67,6 +85,10 @@ $ ./meta get foo.bar
 baz
 $ ./meta get foo.bar --json-value
 "baz"
+$ ./meta lastSuccessfulMeta sd@123:other-job
```

Comment:
meta get --last-successful sd@123:other-job

with `last-successful` being an alias for `external`

Reply:
Did you really want --last-successful or does --external suffice?

Reply:
Made alias - Done